.github/workflows/python_test.yml

```diff
@@ -0,0 +1,48 @@
+name: Python application test
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip3 install -e .
+        sudo curl -sSL http://get.gazebosim.org | sh
+        sudo apt update && sudo apt install python3-tk xvfb
+        git submodule update --init --recursive
+        mkdir build
+        cd build
+        cmake ..
+        make -j
+        cd ..
+        pip3 install -e .
+    - name: Static python syntax checking
+      run: |
+        pip install flake8
+        # static python syntax checking
+        flake8 ./python --count --select=E9,F63,F7,F82 --show-source --statistics
+        # report warnings, ignore E501 line too long
+        flake8 ./python --count --exit-zero --max-complexity=12 --ignore=E501 --statistics
+    - name: Check code style
+      run: |
+        pip3 install pre-commit
+        sudo apt install clang-format-5.0
+        pre-commit install
+        if ! pre-commit run -a ; then
+          git diff
+          exit 0
+        fi
+    - name: Test with pytest
+      run: |
+        cd ./python/social_bot/envs
```

Comment:
"unittest discover" need to run in this path. Let's only run the tests under env for now.